apps/verticalhome/js/app.js

```diff
@@ -199,6 +208,11 @@
         item.updateTitle();
       });
       this.renderGrid();
+
+      app.pinManager.items.forEach(function eachItem(item){
```

Comment:
If I understand correctly, this function ```onLocalized: function() {``` will mostly be called for 'pinApps' view rather than for 'More Apps' and thus it needs probably to take into account currently displayed view, to prevent unnecessary operations.
I think that at least check for currently used mode would be enough. e.g.:
```if (this.inMoreApps) { //work with grid } else { //work with pinManager }```